po/de.po

```diff
@@ -5084,21 +4904,19 @@ msgid "All fonts found.\n"
 msgstr "Fertig. Alle Schriftarten gefunden.\n"
 
 #: ../src/font_file_lister.cpp:211
-#, fuzzy, c-format
-#| msgid "%d fonts could not be found.\n"
+#, c-format
 msgid "One font could not be found\n"
 msgid_plural "%d fonts could not be found.\n"
-msgstr[0] "%d Schriftarten konnten nicht gefunden werden.\n"
+msgstr[0] "Eine Schriftart konnte nicht gefunden worden.\n"
```

Comment:
worden  → werden

*(not sure how i managed to create this new error; maybe i initially reworded it to “Eine Schriftart wurde nicht gefunden.” but reverted it for consistency with the plural and typoed?)*